demo/spec/components/previews/textarea_preview.rb

```diff
@@ -54,6 +54,18 @@ def error
     )
   end
 
+  def page_heading
+    render(
+      CitizensAdviceComponents::Textarea.new(
+        name: "example-textarea-page-heading",
+        label: "Example text area with page heading",
+        options: {
+          page_heading: true
+        }
```

Comment:
In each of these examples do you mind if we include the `hint` option too for completeness?